ddt.py

```diff
@@ -199,7 +199,7 @@ def func(*args):
         )
         return
 
-    with open(data_file_path) as f:
+    with open(data_file_path,coding='utf-8') as f:
```

Comment:
This is failing all tests (plus would fail linting because of the missing space).
Check https://docs.python.org/3/library/functions.html#open for valid argument names...